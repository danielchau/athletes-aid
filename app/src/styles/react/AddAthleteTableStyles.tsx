import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

/**
 * React styles for the Add Athletes Table.
 */
export const addAthleteTableStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            padding: "8px",
            height: "100%",
            display: "flex",
            justifyContent: "center"
        },
        tableContainer: {
            width: "100%",
            height: "100%",
            padding: "8px",
            backgroundColor: "rgba(0, 33, 69, 0.05)",
            overflow: "scroll"
        },
        table: {
            width: "100%"
        },
        tableBodyContainer: {
            width: "2000px",
            overflow: "auto",
            height: "100%"
        },
        tableBody: {
            tableLayout: "fixed"
        },
        tableRow: {
            cursor: "pointer"
        },
        tooltip: {
            backgroundColor: "#fff",
            color: "#4f4f4f",
            boxShadow: theme.shadows[1]
        }
    })
);
