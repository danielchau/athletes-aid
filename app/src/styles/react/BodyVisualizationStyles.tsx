import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

/**
 * React styles for the Body Visualization.
 */
export const bodyVisualizationStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            height: "100%"
        },
        imageContainer: {
            position: "relative",
            alignSelf: "center"
        },
        image: {
            width: "100%"
        },
        dot: {
            height: "15px",
            width: "15px",
            position: "absolute",
            borderRadius: "15px"
        },
        legendContainer: {
            borderLeft: "#dedede solid 1px"
        },
        legendItem: {
            display: "flex",
            color: "rgba(0, 0, 0, 0.54)",
            marginBottom: "2px"
        },
        dotLegend: {
            height: "10px",
            width: "10px",
            borderRadius: "10px",
            marginTop: "6px",
            marginRight: "4px"
        }
    })
);
