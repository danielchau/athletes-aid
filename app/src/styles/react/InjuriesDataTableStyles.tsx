import { makeStyles, Theme, createStyles, lighten } from "@material-ui/core/styles";

/**
 * React styles for the Injury Data Table.
 */
export const injuriesDataTableStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            marginTop: theme.spacing(3)
        },
        paper: {
            width: "100%",
            marginBottom: theme.spacing(2),
            backgroundColor: "rgba(0, 33, 69, 0.05)"
        },
        table: {
            minWidth: 750
        },
        tableWrapper: {
            overflowX: "auto"
        },
        tableRow: {
            cursor: "pointer"
        },
        visuallyHidden: {
            border: 0,
            clip: "rect(0 0 0 0)",
            height: 1,
            margin: -1,
            overflow: "hidden",
            padding: 0,
            position: "absolute",
            top: 20,
            width: 1
        }
    })
);

/**
 * React styles for the Injury Data Table Toolbar Header.
 */
export const injuriesDataTableToolbarStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            paddingLeft: theme.spacing(2),
            paddingRight: theme.spacing(1)
        },
        highlight:
            theme.palette.type === "light"
                ? {
                      color: theme.palette.primary.main,
                      backgroundColor: lighten(theme.palette.primary.light, 0.85)
                  }
                : {
                      color: theme.palette.text.primary,
                      backgroundColor: theme.palette.secondary.dark
                  },
        title: {
            flex: "1 1 100%"
        },
        export: {
            margin: "auto",
            color: theme.palette.secondary.main,
            display: "flex"
        }
    })
);
