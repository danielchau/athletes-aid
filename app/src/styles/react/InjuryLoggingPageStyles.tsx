import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

export const injuryLoggingPageStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            padding: "8px"
        },
        backButton: {
            marginRight: theme.spacing(1)
        },
        completedContainer: {
            height: "calc(100% - 116px)",
            textAlign: "center",
            paddingTop: "20%"
        },
        instructions: {
            marginTop: theme.spacing(1),
            marginBottom: theme.spacing(1)
        },
        paper: {
            color: theme.palette.text.secondary,
            backgroundColor: "#fafafa"
        },
        paperContent: {
            height: "100%",
            padding: "16px",
            paddingRight: "8px",
            overflow: "auto"
        },
        stepper: {
            backgroundColor: "#fafafa",
            borderRadius: "4px"
        },
        loggingContent: {
            height: "calc(100% - 168px)",
            marginBottom: "52px",
            marginTop: "8px",
            overflow: "hidden"
        },
        loggingBottomButtons: {
            position: "fixed",
            bottom: 0,
            right: 0,
            height: "52px",
            padding: "8px"
        },
        multilineInput: {
            width: "100%"
        }
    })
);
