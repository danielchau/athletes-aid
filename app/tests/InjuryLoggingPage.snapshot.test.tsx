import * as React from 'react'
import InjuryLoggingPage from '../src/components/InjuryLoggingPage'
import renderer from 'react-test-renderer'

const mockTeam = {
    name: "Test name",
    athletes: [{
        id: "Test id",
        name: "Test name"
    }]
}

it('InjuryDialog renders correctly', () => {
  const tree = renderer.create(<InjuryLoggingPage selectedTeam={mockTeam} existingInjury={null} callbackUponFinishing={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})