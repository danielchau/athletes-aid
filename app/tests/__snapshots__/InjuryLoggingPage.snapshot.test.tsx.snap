// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InjuryDialog renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 makeStyles-paper-5 MuiPaper-rounded"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation0 MuiStepper-root MuiStepper-horizontal makeStyles-stepper-7 MuiStepper-alternativeLabel"
    >
      <div
        className="MuiStep-root MuiStep-horizontal MuiStep-alternativeLabel"
      >
        <span
          className="MuiStepLabel-root MuiStepLabel-horizontal MuiStepLabel-alternativeLabel"
        >
          <span
            className="MuiStepLabel-iconContainer MuiStepLabel-alternativeLabel"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiStepIcon-root MuiStepIcon-active"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <circle
                cx="12"
                cy="12"
                r="12"
              />
              <text
                className="MuiStepIcon-text"
                textAnchor="middle"
                x="12"
                y="16"
              >
                1
              </text>
            </svg>
          </span>
          <span
            className="MuiStepLabel-labelContainer"
          >
            <span
              className="MuiTypography-root MuiStepLabel-label MuiStepLabel-alternativeLabel MuiStepLabel-active MuiTypography-body2 MuiTypography-displayBlock"
            >
              Injury Details
            </span>
          </span>
        </span>
      </div>
      <div
        className="MuiStep-root MuiStep-horizontal MuiStep-alternativeLabel"
      >
        <div
          className="MuiStepConnector-root MuiStepConnector-horizontal MuiStepConnector-alternativeLabel Mui-disabled"
        >
          <span
            className="MuiStepConnector-line MuiStepConnector-lineHorizontal"
          />
        </div>
        <span
          className="MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled MuiStepLabel-alternativeLabel"
        >
          <span
            className="MuiStepLabel-iconContainer MuiStepLabel-alternativeLabel"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiStepIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <circle
                cx="12"
                cy="12"
                r="12"
              />
              <text
                className="MuiStepIcon-text"
                textAnchor="middle"
                x="12"
                y="16"
              >
                2
              </text>
            </svg>
          </span>
          <span
            className="MuiStepLabel-labelContainer"
          >
            <span
              className="MuiTypography-root MuiStepLabel-label MuiStepLabel-alternativeLabel MuiTypography-body2 MuiTypography-displayBlock"
            >
              Further Details
            </span>
          </span>
        </span>
      </div>
      <div
        className="MuiStep-root MuiStep-horizontal MuiStep-alternativeLabel"
      >
        <div
          className="MuiStepConnector-root MuiStepConnector-horizontal MuiStepConnector-alternativeLabel Mui-disabled"
        >
          <span
            className="MuiStepConnector-line MuiStepConnector-lineHorizontal"
          />
        </div>
        <span
          className="MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled MuiStepLabel-alternativeLabel"
        >
          <span
            className="MuiStepLabel-iconContainer MuiStepLabel-alternativeLabel"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiStepIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <circle
                cx="12"
                cy="12"
                r="12"
              />
              <text
                className="MuiStepIcon-text"
                textAnchor="middle"
                x="12"
                y="16"
              >
                3
              </text>
            </svg>
          </span>
          <span
            className="MuiStepLabel-labelContainer"
          >
            <span
              className="MuiTypography-root MuiStepLabel-label MuiStepLabel-alternativeLabel MuiTypography-body2 MuiTypography-displayBlock"
            >
              Review
            </span>
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    className="makeStyles-loggingContent-8"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 makeStyles-paperContent-6 MuiPaper-rounded"
    >
      <p
        className="MuiTypography-root makeStyles-instructions-117 MuiTypography-body1"
      >
        Please input the following information regarding the injury.
      </p>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
          data-shrink={true}
          id="team-select-label"
        >
          Team Name
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            aria-labelledby=" team-select"
            className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSelect MuiOutlinedInput-inputSelect"
            id="team-select"
            onBlur={[Function]}
            onClick={null}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            Test name
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
            value="Test name"
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 8,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 75.5,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <div
          aria-expanded={false}
          aria-owns={null}
          className="MuiAutocomplete-root"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          role="combobox"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
              data-shrink={false}
              htmlFor="athlete-select"
              id="athlete-select-label"
            >
              Athlete Name
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiAutocomplete-inputRoot MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
              onClick={[Function]}
            >
              <input
                aria-activedescendant={null}
                aria-autocomplete="list"
                aria-controls={null}
                aria-invalid={false}
                autoCapitalize="none"
                autoComplete="off"
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
                defaultValue=""
                disabled={false}
                id="athlete-select"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onMouseDown={[Function]}
                required={false}
                spellCheck="false"
                type="text"
                value=""
              />
              <button
                aria-label="Clear"
                className="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-clearIndicator"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={-1}
                title="Clear"
                type="button"
              >
                <span
                  className="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                    />
                  </svg>
                </span>
              </button>
              <button
                aria-label="Open popup"
                className="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-popupIndicator"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={-1}
                title="Open popup"
                type="button"
              >
                <span
                  className="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                </span>
              </button>
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
                style={
                  Object {
                    "paddingLeft": 8,
                  }
                }
              >
                <legend
                  className="PrivateNotchedOutline-legend-194"
                  style={
                    Object {
                      "width": 0.01,
                    }
                  }
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "&#8203;",
                      }
                    }
                  />
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root makeStyles-dateInput-115 MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
          data-shrink={true}
          htmlFor="date-select"
          id="date-select-label"
        >
          Injury Date
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
            disabled={false}
            id="date-select"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
            value="11/24/2019"
          />
          <div
            className="MuiInputAdornment-root MuiInputAdornment-positionEnd"
          >
            <button
              className="MuiButtonBase-root MuiIconButton-root"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
              </span>
            </button>
          </div>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 8,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 0,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
        
      </div>
      <label
        className="MuiFormControlLabel-root makeStyles-checkboxInput-116"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-246 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <input
              className="PrivateSwitchBase-input-249"
              data-indeterminate={false}
              disabled={false}
              onChange={[Function]}
              type="checkbox"
              value={false}
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
              />
            </svg>
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Sports Related
        </span>
      </label>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink={false}
          id="event-type-select-label"
        >
          Event Type
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            aria-labelledby=" event-type-select"
            className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSelect MuiOutlinedInput-inputSelect"
            id="event-type-select"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
            value=""
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 42,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink={false}
          id="position-select-label"
        >
          Position
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            aria-labelledby=" position-select"
            className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSelect MuiOutlinedInput-inputSelect"
            id="position-select"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
            value=""
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 34.5,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink={false}
          id="side-of-body-select-label"
        >
          Side Of Body
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            aria-labelledby=" side-of-body-select"
            className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSelect MuiOutlinedInput-inputSelect"
            id="side-of-body-select"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
            value=""
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 49.5,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink={false}
          id="body-location-select-label"
        >
          Location On Body
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            aria-labelledby=" body-location-select"
            className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSelect MuiOutlinedInput-inputSelect"
            id="body-location-select"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
            value=""
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 60.75,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink={false}
          id="injury-type-select-label"
        >
          Injury Type
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            aria-labelledby=" injury-type-select"
            className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSelect MuiOutlinedInput-inputSelect"
            id="injury-type-select"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
            value=""
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 42,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root makeStyles-formControl-112"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
          data-shrink={true}
          id="severity-select-label"
        >
          Severity
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            aria-labelledby=" severity-select"
            className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSelect MuiOutlinedInput-inputSelect"
            id="severity-select"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            0
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
            value={0}
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-193 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 8,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-194"
              style={
                Object {
                  "width": 53,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <div
      className="makeStyles-loggingBottomButtons-9"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-backButton-2 Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
            />
          </svg>
          Back
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Next
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
            />
          </svg>
        </span>
      </button>
    </div>
  </div>
</div>
`;
